ui/app/src/utils/state.ts

```diff
@@ -35,22 +35,6 @@ export function setStoredPreviewChoice(site: string, value: string) {
   return window.localStorage.setItem(`craftercms.previewCompatChoice.${site}`, value);
 }
 
-export function getStoredEditModeChoice(): string {
-  return window.localStorage.getItem(`craftercms.editModeChoice`);
-}
-
-export function setStoredEditModeChoice(value: string) {
-  return window.localStorage.setItem(`craftercms.editModeChoice`, value);
-}
-
-export function getStoredhighlightModeChoice(): string {
-  return window.localStorage.getItem(`craftercms.highlightModeChoice`);
-}
-
-export function setStoredhighlightModeChoice(value: string) {
-  return window.localStorage.setItem(`craftercms.highlightModeChoice`, value);
-}
```

Comment:
We still need these keys as constants somewhere.

Reply:
to use where? to set preference im just sending  { editMode,  highlightMode, ...etc }

Reply:
To set the user property. The key with which you will call the setProperties service.